blueocean-core-js/src/js/config.js

```diff
@@ -65,12 +65,13 @@ export default {
     showOrg() {
         return this.isFeatureEnabled('organizations.enabled', false);
     },
-
+    
     getJenkinsRootURL() {
         if (!config.isLoaded) {
             this.loadUrls();
         }
-        return config.jenkinsRootURL || '/jenkins';
```

Comment:
this will be set to '' in the case of it running on root (from jenkins), so this never would have worked.